### DISCUSSION TITLE
“简单主页”提示403拒绝访问

### DISCUSSION THREAD

**Discussion by WuHuX** [Meloong-Git/PCL#5126] [自定义主页]
我想这不是我的问题，因为我试了自己的3台设备，也让其他地区的朋友试了，都出现403
错误提示：联网下载自定义主页失败(https://gitee.com/mfn233/PCL-Mainpage/raw/main/Custom.xaml):获取结果失
败(7，远程服务器返回错误:(403)已禁止。，https://gitee.com/mfn233/PCL-Mainpage/raw/main/
Custom.xaml)
→远程服务器返回错误:(403)已禁止。
在 PCL.ModNet.NetGetCodeByRequestRetry(String Url, Encoding Encode, String Accept, Boolean IsJson,
String BackupUrl, Boolean UseBrowserUserAgent)
在PCL.PageLaunchRight.OnlineLoaderSub(LoaderTask`2 Task)
在 PCL.ModNet.NetGetCodeByRequestOnce(String Url, Encoding Encode, Int32 Timeout, Boolean IsJson,
String Accept, Boolean UseBrowserUserAgent)
错误类型:System.Net.WebException

**Comment by Ad-closeNN** [comment]
- #5012

请使用如下链接并填写至 ![image](https://github.com/user-attachments/assets/cda448b8-8aec-49d2-8512-f6729d45b524)

```
https://raw.gitcode.com/MFn233/PCL-Mainpage/raw/main/Custom.xaml
```
